config.go

```diff
@@ -56,11 +58,12 @@ type DatabaseConfig struct {
 	Pass      string
 	Collation string
 	Params    map[string]string
-	// SQL query comments to differentiate Ghostferry's binlog events
-	// Optional: defaults to empty string (no comments)
-	Marginalia string
+	TLS       *TLSConfig
 
-	TLS *TLSConfig
+	// SQL annotations used to differentiate Ghostferry's DMLs
```

Comment:
Will be providing a comment with further context when adding the Target verification